.github/CODEOWNERS

```
@@ -1 +1,2 @@
 * @actions/actions-service
+* @actions/runner-images-team
```

```suggestion
* @actions/setup-actions-team 
```

Should this be `setup-actions-team`?  I saw that used somewhere else.  Also, I think it's time to get rid of `actions-service` as codeowner.